.cascade-code/Chapman.edu/_cascade/blocks/html/global_footer.html

```diff
@@ -59,14 +59,70 @@ <h2>Services</h2>
             <h2>Social</h2>
           </div>
           <ul class="link-list social-icon-list">
-            <li><a class="icon icon-twitter" href="http://twitter.com/#!/chapmanu" rel="noopener noreferrer" target="_blank"><span class="sr-only">Twitter</span></a></li>
-            <li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" rel="noopener noreferrer" target="_blank"><span class="sr-only">Facebook</span></a></li>
-            <li><a class="icon icon-pinterest" href="http://pinterest.com/chapmanu/" rel="noopener noreferrer" target="_blank"><span class="sr-only">Pinterest</span></a></li>
-            <li><a class="icon icon-graduation" href="http://itunes.apple.com/us/institution/chapman-university/id430678922" rel="noopener noreferrer" target="_blank"><span class="sr-only">iTunes</span></a></li>
-            <li><a class="icon icon-youtube" href="http://www.youtube.com/user/ChapmanUniversity" rel="noopener noreferrer" target="_blank"><span class="sr-only">youTube</span></a></li>
-            <li><a class="icon icon-linkedin2" href="http://www.linkedin.com/company/10701?trk=tyah" rel="noopener noreferrer" target="_blank"><span class="sr-only">LinkedIn</span></a></li>
-            <li><a class="icon icon-google-plus" href="https://plus.google.com/112412382370481463024/posts" rel="noopener noreferrer" target="_blank"><span class="sr-only">Google Plus</span></a></li>
-            <li><a class="icon icon-instagram" href="http://www.instagram.com/chapmanu" rel="noopener noreferrer" target="_blank"><span class="sr-only">Instagram</span></a></li>
+            <li>
+              <a href="https://www.facebook.com/ChapmanUniversity" target="_blank">
+                <span class="sr-only">Facebook</span>
+                <svg class="icon icon-facebook" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 32 32">
+                  <path d="M19 6h5v-6h-5c-3.9 0-7 3.1-7 7v3h-4v6h4v16h6v-16h5l1-6h-6v-3c0-0.5 0.5-1 1-1z"></path>
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com/#!/chapmanu" target="_blank">
+                <span class="sr-only">Twitter</span>
+                <svg class="icon icon-twitter" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 32 32">
```

Comment:
The "icon" and "icon-{name}" classes are related only to icomoon and should be removed.